subgraph/generated/FleekNFA/FleekNFA.ts

```diff
@@ -0,0 +1,1306 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+
+import {
+  ethereum,
+  JSONValue,
+  TypedMap,
+  Entity,
+  Bytes,
+  Address,
+  BigInt,
+} from '@graphprotocol/graph-ts';
+
+export class Approval extends ethereum.Event {
+  get params(): Approval__Params {
+    return new Approval__Params(this);
+  }
+}
+
+export class Approval__Params {
```

Comment:
When does one use this file vs the other TS file in the query-examples folder?

Reply:
This file is used in the src dir and query-examples is isolated from the other folders. The main.ts file in there is written by me.

Reply:
Thank you for the explanation.  To summarize, the ones in the query-examples are generated by the graph client generator and are meant to be used by any clients of thegraph/this subgraph.  Meanwhile the ones in our generated folder is TS wrappers around the schema and events.